patterns/alz/alz-portal.json

```diff
@@ -0,0 +1,1090 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2021-09-09/uiFormDefinition.schema.json#",
+  "view": {
+    "kind": "Form",
+    "properties": {
+      "title": "Azure Monitor Baseline Alerts Accelerator",
```

Comment:
- [ ] Can you add "[Preview] -" to the title.
- [ ] Please add ALZ to the name for example "AMBA-ALZ Accelerator". To prevent any misunderstandings as this only deploys the ALZ-Pattern.